projects/aca-content/assets/app.extensions.json

```diff
@@ -306,7 +307,7 @@
           {
             "id": "app.create.fileFromTemplate",
             "order": 300,
-            "icon": "description",
+            "icon": "adf:file_from_template",
```

Comment:
missing icon

Reply:
this has been changed to use pre-existing icons